Salwador/View/ContentView.swift

```diff
@@ -0,0 +1,144 @@
+//
+//  ContentView.swift
+//  Salwador
+//
+//  Created by Ailiniyazi Maimaiti on 31.01.23.
+//
+
+import SwiftUI
+import UIKit
+
+struct ContentView: View {
+    @State private var prompText: String = ""
+    @State private var buttonText: String = "Ready?"
+    @State private var imagePath: String = "salvador-man"
+    @State private var image: UIImage? = nil
+    @State private var isLoading: Bool = false
+
+    var body: some View {
+        ZStack {
+            Color(red: 0.0, green: 0.4666666666666667, blue: 0.7137254901960784)
+
+            VStack {
+                appNameView
+                Spacer()
+                textEditorView
+                submitButtonView
+                Spacer()
+                imageView
+                Spacer()
+            }
+        }
+        .padding()
+    }
```

Comment:
I really like that you splitted the view in subviews! 👍 Nevertheless I have some suggestion for improvements:

I would remove the `ZStack` and use the `.background` modifier to add the `Color` there. Like this:

```suggestion
    var body: some View {
        VStack {
            appNameView
            Spacer()
            textEditorView
            submitButtonView
            Spacer()
            imageView
            Spacer()
        }
        .padding()
        .background(Color(red: 0.0, green: 0.4666666666666667, blue: 0.7137254901960784))
    }
```

You could also think about adding the Color the the `Assets.xcassets` catalog and load it from there instead of defining it hardcoded in the view code.